Mathlib/Algebra/Group/Defs.lean

```diff
@@ -808,7 +808,7 @@ independent:
 * Without `DivisionMonoid.div_eq_mul_inv`, you can define `/` arbitrarily.
 * Without `DivisionMonoid.inv_inv`, you can consider `WithTop Unit` with `a⁻¹ = ⊤` for all `a`.
 * Without `DivisionMonoid.mul_inv_rev`, you can consider `WithTop α` with `a⁻¹ = a` for all `a`
-  where `α` non commutative.
+  where `α` noncommutative.
```

Comment:
bikeshed: I would hyphenate this - can you at least make sure to follow the local style in this file? Thanks!

Reply:
See the general comment below.

In the rest of the file there are two cases of "non-zero", so if I want to align with those I should probably hyphenate, but I still feel the considerations I mention in the general comment weighs higher.